statefun-flink/statefun-flink-core/src/main/java/org/apache/flink/statefun/flink/core/httpfn/DefaultHttpRequestReplyClientSpec.java

```diff
@@ -123,6 +131,16 @@ private static Duration requireNonZeroDuration(Duration duration) {
     }
   }
 
+  private static final class DurationJsonSerialize extends JsonSerializer<Duration> {
```

Comment:
We may need to backport this commit. I'm not sure how RemoteFunctionBuilder ever worked...